src/components/UI/docs/DocumentNav.tsx

```diff
@@ -20,7 +20,7 @@ export const DocumentNav: FC<Props> = ({ content }) => {
   const activeHash = useActiveHash(parsedHeadings.map(heading => heading!.headingId));
 
   return parsedHeadings.length ? (
-    <Box as='aside' position='sticky' top='4'>
+    <Box as='aside' position='sticky' h='95vh' overflowY='auto' top='4'>
```

Comment:
Not critical, but might be worth switching to a `calc` function here to subtract a static amount, such as `3rem`, from `100vh`

```suggestion
    <Box as='aside' position='sticky' h='calc(100vh - 3rem)' overflowY='auto' top='4'>
```